atomic_reactor/plugins/exit_koji_promote.py

```diff
@@ -460,10 +460,20 @@ def add_log_type(output):
                 },
             },
         })
-        annotations = self.workflow.build_result.annotations
-        if annotations and 'digests' in annotations:
-            digests = get_digests_map_from_annotations(annotations['digests'])
-            metadata['extra']['docker']['digests'] = digests
+
+        if self.workflow.push_conf.docker_registries:
+            if self.workflow.tag_conf.unique_images:
+                image = self.workflow.tag_conf.unique_images[0]
```

Comment:
I'm trying to work out how this works. 

For arrangement >= 3:
1. In worker build, the 'digests' annotation is added to the OpenShift worker build via `store_metadata_in_osv3` plugin.
2. In orchestrator build, the `orchestrate_build` plugin aggregates the annotation from each worker build under the `worker-builds` annotation.
3. `koji_import` uses the `digests` annotation of each worker build to create the aggregated desired digests map.

For prod builds, I think we want to iterate over digests the same way that `store_metadata_in_osv3` does. Shouldn't we then loop for each `docker_registry` and `each tag_conf.images`?

Reply:
ok I'll iterate through each docker_registry.
even though right now there is just one registry, it is possible to have more, and even though the digests for same tag would be same, it is possible that each registry will provide different set of digest versions, so to collect all versions I'll loop

Reply:
as for looping through images, as of now we have only 1 unique tag, and digests as of now contains digests for 1 tag only